### PR TITLE
feat: add artela testnet config to chain registry.

### PR DESCRIPTION
**Pull request by 0xYuan90**
feat: add artela testnet config to chain registry.

Artela Website: https://artela.network
Artela Twitter:  https://twitter.com/Artela_Network

### CODE REVIEW

File: testnets/artelatestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          "denom": "art",
+          "exponent": 18
+        }
+      ],
+      "base": "uart",
+      "name": "Artela Testnet",
+      "display": "art",
+      "symbol": "ART",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/artelatestnet/images/artela.png"
```
Comment (JeremyParish69) [inline testnets/artelatestnet/assetlist.json:22]: Do you have a square aspect ratio token logo?

Comment (JeremyParish69) [reply testnets/artelatestnet/assetlist.json:22]: The current image should be used for a chain logo defined in chain.json (but not defined, currently)

Comment (0xYuan90) [reply testnets/artelatestnet/assetlist.json:22]: thanks Jeremy, I will add new logo

Comment (0xYuan90) [reply testnets/artelatestnet/assetlist.json:22]: hi Jeremy, new logo has updated

Comment (0xYuan90) [reply testnets/artelatestnet/assetlist.json:22]: @JeremyParish69 hi Jeremy, new logo has updated, thanks, please review

**Pull request merged**